tox.ini

```diff
@@ -10,14 +10,17 @@ setenv = HOME = {envtmpdir}
 commands =
   mitmdump --version
   pytest --timeout 60 -vv --cov-report xml \
-    --cov=mitmproxy --cov=pathod --cov=release \
-    --full-cov=mitmproxy/ --full-cov=pathod/ \
+    --continue-on-collection-errors \
+    --cov=mitmproxy --cov=release \
+    --full-cov=mitmproxy/ \
     {posargs}
 
 [testenv:flake8]
-deps = flake8==3.8.4
+deps =
+  flake8>=3.8.4,<4
+  flake8-tidy-imports>=4.2.0,<5
```

Comment:
Is this actually useful? I don't see anything in https://pypi.org/project/flake8-tidy-imports/ that seems particularly useful to catch.